### ISSUE TITLE
error versus reject curves

### ISSUE THREAD

**Issue by duxi** [pterhoer/FaceImageQuality#33]
Great job and novel idea! i'm trying to test it on our private dataset and wandering if there is some hint(or code) for the error versus reject curves. Thanks!

**Comment by pterhoer** [comment]
Hi Duxi,

thank you for your interest in our work! And sorry for the late reply...

Currently, I am not aware of how much work it is to make the code for the error-vs-reject curves public available. (At the moment it is optimized for our hardware)
But the general idea is not too complicated:
1. Determine the face image quality of each image in your dataset.
2. Create a new dataset D_x by rejecting x% of the images with the lowest quality scores.
3. Compute the face recognition error on D_x. E.g. by using the roc_curve method from sklearn (sklearn.metrics.roc_curve).
Now you have one point of the error-vs-reject curve at x% on the x-axis. To make the curve more clean, just repeat step 2 and 3 with different x values. 

Best
Philipp
